modules/@apostrophecms/modal/ui/apos/mixins/AposEditorMixin.js

```diff
@@ -94,7 +92,12 @@ export default {
               ...Object.fromEntries(results)
             };
           } catch (error) {
-            console.error(error);
+            await apos.notify(this.$t('apostrophe:errorEvaluatingExternalCondition', { name: field.name }), {
+              type: 'danger',
+              icon: 'alert-circle-icon',
+              dismiss: true,
+              localize: false
```

Comment:
not sure about this one

Reply:
Accepting for now but it would make more sense to take the label (which must be localized) as the parameter.